guides/common/modules/proc_upgrading-hosts-to-next-major-release.adoc

```diff
@@ -63,5 +80,9 @@ Select the issue.
 Repeat this step for other issues.
 .. After you selected any issues with remediation commands, click *Fix Selected* and submit the job.
 .. After the issues are fixed, click *Rerun*, and then click *Submit* to run the pre-upgrade check again to verify that the hosts you are upgrading do not have any issues and are ready to be upgraded.
-. If the pre-upgrade check verifies that the hosts do not have any issues, click *Run Upgrade* and click *Submit* to start the upgrade.
-Alternatively, you can upgrade your host by selecting *Upgrade with Leapp* in the *Schedule Remote Job* drop down menu.
+. Start the upgrade:
+.. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+.. Select the hosts that you want to upgrade.
+.. If you want to upgrade to the latest available release version, select *Upgrade with Leapp* from the *Schedule a Job* drop-down menu.
+.. If you want to upgrade to a specific release version, click *Schedule a job*, and select *Leapp - Upgrade* from the *Job category* list.
```

Comment:
Reversed logic. Compare these lines 86-87 with 56-57.

It should be like this:
... upgrade to the latest available release version -> select *Leapp - Upgrade*  ...
... upgrade to a specific release version ->  click *Schedule a job* ...

Reply:
Updated both sets of steps to ensure consistency.